packages/nvidia_nat_mcp/scripts/check_mcp_auth_jwt.py

```
@@ -0,0 +1,340 @@
+            state_key = oauth_state if isinstance(oauth_state, str) else "<none>"
+            if isinstance(auth_url, str) and state_key not in opened_oauth_states:
+                webbrowser.open(auth_url)
+                opened_oauth_states.add(state_key)
+        elif status == "interaction_required":
+            print(json.dumps(current_payload, indent=2))
+            return
+
+        done, failed = _handle_execution_status_payload(current_payload)
+        if done:
+            if failed:
+                raise RuntimeError("Interactive HTTP execution failed.")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)